config.ru

```diff
@@ -2,7 +2,10 @@
 
 require 'rack'
 require './lib/foreman/telemetry_rack'
```

Comment:
There are still things in here. With this patch I get
```
No such file to load -- prometheus/client.rb (LoadError)
  /usr/share/foreman/vendor/ruby/2.3.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require&apos;
  /usr/share/foreman/vendor/ruby/2.3.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require&apos;
  /usr/share/foreman/vendor/ruby/2.3.0/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:292:in `block in require&apos;
  /usr/share/foreman/vendor/ruby/2.3.0/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:258:in `load_dependency&apos;
  /usr/share/foreman/vendor/ruby/2.3.0/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:292:in `require&apos;
  /usr/share/foreman/lib/foreman/telemetry_sinks/prometheus_sink.rb:1:in `<top (required)>&apos;
[...]
```

Reply:
Odd, I was able to start the dev server.

Reply:
I can reproduce this if I add a ```:telemetry``` group to the ```settings.yaml```